test/test_field.jl

```diff
@@ -471,6 +471,20 @@ end
 
         for arch in archs, FT in float_types
             run_field_reduction_tests(FT, arch)
+
+            @info "    Test reductions on WindowedFields..."
+
+            grid = RectilinearGrid(arch, FT, size=(2, 3, 4), x=(0, 1), y=(0, 1), z=(0, 1))
+            c = CenterField(grid)
+            Random.seed!(42)
+            set!(c, rand(size(c)...))

```

Comment:
PS @navidcy this line used to use broadcasting, eg ` c .= ...`. Note that broadcasting with Fields has limited support, so probably not good for testing (or anywhere really... would be nice to make it better but there's lots to do...)

Reply:
I think I took it from the doctest. I’ll change it there!